hudi-spark/src/main/scala/org/apache/hudi/HoodieSparkSqlWriter.scala

```diff
@@ -376,31 +379,35 @@ private[hudi] object HoodieSparkSqlWriter {
     metaSyncSuccess
   }
 
+  /**
+   * Scala says method cannot have more than 7 arguments. So group all table/action specific information into a case class.
```

Comment:
please remove the scala specific comment. its good to encapsulate like this anyway